tests/unit/dataactvalidator/test_a34_appropriations.py

```diff
@@ -8,8 +8,8 @@
 
 
 def test_column_headers(database):
-    expected_subset = {'row_number', 'budget_authority_unobligat_fyb', 'expected_value_GTAS SF133 Line 2490',
-                       'difference'}
+    expected_subset = {'uniqueid_TAS', 'row_number', 'budget_authority_unobligat_fyb',
+                       'expected_value_GTAS SF133 Line 2490', 'difference'}
     actual = set(query_columns(_FILE, database))
     assert (actual & expected_subset) == expected_subset
```

Comment:
Is there a reason to leave any of the tests that use these bitwise operators as having them? They seem pretty redundant if we're already making sure it's `==`. We can probably just do `actual == expected_subset` for all of them.

Since I can't comment on all the spots that have this, here's a quick list for easy access: A32, A33.2, A34, A36